internal/impl/ollama/base_processor.go

```diff
@@ -69,10 +68,6 @@ func commonFields() []*service.ConfigField {
 				Optional().
 				Advanced().
 				Description("Map the model into memory. This is only support on unix systems and allows loading only the necessary parts of the model as needed."),
-			service.NewBoolField(bopFieldUseMLock).
```

Comment:
Might be worth leaving this in place as `deprecated` to avoid breaking configs where people just set it blindly.